modules/minimum-required-permissions-ipi-gcp.adoc

```diff
@@ -140,13 +155,18 @@ If your organization’s security policies require a more restrictive set of per
 * `compute.httpHealthChecks.get`
 * `compute.httpHealthChecks.list`
 * `compute.httpHealthChecks.useReadOnly`
+* `compute.regionHealthCheckServices.list`
```

Comment:
During [my testing](https://issues.redhat.com/browse/OCPBUGS-34129?focusedId=24938938&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-24938938), the permission is not required.

Reply:
Sorry for the late reply, yes, the permission `compute.regionHealthCheckServices.list` is not needed for both create and destroy, I'm saying this because the service account I used for the minimal permissions testing doesn't have the permission and both "create cluster" and "destroy cluster" work well. Thanks!